Northstar.Client/mod/scripts/vscripts/ui/menu_ingame.nut

```diff
@@ -85,10 +85,12 @@ void function InitInGameMPMenu()
 	var gameHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_GAME" )
 	var leaveButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#LEAVE_MATCH" )
 	Hud_AddEventHandler( leaveButton, UIE_CLICK, OnLeaveButton_Activate )
-	#if DEV
-		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "Dev" )
-		Hud_AddEventHandler( devButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "DevMenu" ) ) )
-	#endif
+	
+	if ( GetConVarBool( "ns_allow_team_change" ) )
+	{
+		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#SWITCH_TEAMS" )
```

Comment:
I *believe* that there is an overload of this function that can take a callback for if the button is locked or not, might be better to use that and simple disable the button instead of fully hiding it?

Reply:
Yeah gonna use `Hud_SetLocked( devButton, true )` instead of hiding, i actually forgot that's an option for buttons.